dali/kernels/test/warp_test/warp_gpu_test.cu

```diff
@@ -41,8 +41,9 @@ class WarpPrivateTest {
   }
 };
 
-void IsWarpKernelValid() {
+TEST(WarpGPU, check_kernel) {
   check_kernel<WarpGPU<AffineMapping2D, 2, float, uint8_t, float>>();
+  SUCCEED();
```

Comment:
unless this is testing anything. I'd say remove it

Reply:
That's the way @szalpal wanted it and I'm really fed up with arguing about such details...

Reply:
I convinced him to get rid of it in another PR. Anyway, won't push more on this